Rakefile

```diff
@@ -3,4 +3,12 @@
 
 require File.expand_path('../config/application', __FILE__)
 
+if Rails.env.development? || Rails.env.test?
+  require 'rubocop/rake_task'
+
+  RuboCop::RakeTask.new
+end
+
+task(:default).prerequisites << task(:rubocop)
```

Comment:
The default action for `rake` command is launch specs, and if you try to do it in production mode...

```bash
rake RAILS_ENV=production
Abort testing: Your Rails environment is running in production mode!
```